.github/workflows/deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Push openml-java packages to maven central repository
+
+on:
+  push:
+    tags:
+      - '*.*.*'
```

Comment:
We should also allow for experimental versions, either based on the classic `xp-*` branch glob, or the `*.*.*-*` tag (which is actually the one we're using above, so we might just go with that one)